src/main/java/org/sagebionetworks/web/client/widget/table/v2/results/cell/EntityIdCellRenderer.java

```diff
@@ -22,6 +22,7 @@
 	ClickHandler customClickHandler;
 	CellAddress address;
 	boolean hideIfLoadError;
+	boolean showEntityId = false;

```

Comment:
By default, each EntityIdCellRenderer instance starts out with showEntityId = false.  But you can flip it to true (before setValue()!) to show it in the output link text